src/main/java/com/systemmeltdown/robotlib/subsystems/drive/SkidSteerDriveSubsystem.java

```diff
@@ -37,6 +37,10 @@
 	private int m_clicksPerInch = 0;
 	private int m_maxSpeedClicksPerSecond = 0;
 
+	public int getClicksPerInch() {
```

Comment:
Let's not expose any public methods based on clicks. Let's use real measurable values for our public interfaces.